objects/Object.java

```diff
@@ -0,0 +1,54 @@
+package objects;
```

Comment:
This is implementing the game using inheritance. Can you kindly split the pull requests so it is easier to review?

Reply:
I have created a separate PR for this. #12